ns1/ipam.py

```diff
@@ -347,14 +365,7 @@ def success(result, *args):
                 return self
 
         if self.id is None:
-            if self.dhcp4 is None or self.dhcp6 is None or self.name is None or self.service_group_id is None:
```

Comment:
Removed possibility of searching by name, ID is the only viable search criteria.